### PR TITLE
Git Gut - v1.0.0 Touch Up

### PR DESCRIPTION
**Pull request by NyanKyaw**
## Details

General review of the existing content on `dev`. Most changes were made to fix spelling/grammar issues.

### CODE REVIEW

File: .gitignore
Comment (oraqlle) [inline .gitignore:1]: Make sure to add empty line at the end of a file, it ensure the files are compatible across Git on Windows and UNIX (Linux, MacOS, FreeBSD etc.).

File: src/github/licenses-oss.md
Comment (oraqlle) [inline src/github/licenses-oss.md:1]: For the links to the licenses, you can prefix the link (the pit in parenthesis) with this <https://github.com/MonashDeepNeuron/Template-Docs/tree/main/>. This is what I was supposed to do but I don't think I had made the repo yet 🤣.

**Review by oraqlle — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (NyanKyaw) [reply .gitignore:1]: Yep done. Thanks for letting me know about this.

File: src/github/licenses-oss.md
Comment (NyanKyaw) [reply src/github/licenses-oss.md:1]: All links added, thanks.

**Review by oraqlle — APPROVED** [review]
Looks good, thanks @NyanKyaw.

**Pull request merged**